### PR TITLE
fix amp-live-list types

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
@cramforce PTAL

File: extensions/amp-live-list/0.1/amp-live-list.js [hunk 1]
```diff
@@ -699,11 +735,14 @@ export class AmpLiveList extends AMP.BaseElement {
    * Iterates over the child elements from first to last and invokes the
    * callback with the current child element passed in as the first argument.
    *
-   * @param {!Element} parent
+   * @param {?Element} parent
    * @param {function(!Element)} cb
    * @private
    */
   eachChildElement_(parent, cb) {
+    if (!parent) {
```
Comment (cramforce) [inline extensions/amp-live-list/0.1/amp-live-list.js]: Can you explain why these might be null? Or should it be handled by the caller?

Comment (erwinmombay) [inline extensions/amp-live-list/0.1/amp-live-list.js]: it probably makes more sense for it to be handled by the caller. the only reason why these might be null are the values supplied on the call site are all elements that are only ready during buildCallback

Comment (cramforce) [inline extensions/amp-live-list/0.1/amp-live-list.js]: Then lets move to caller. `assertElement` should be perfect.

Comment (erwinmombay) [inline extensions/amp-live-list/0.1/amp-live-list.js]: @cramforce done. PTAL again

**Comment by cramforce** [comment]
LGTM

**Pull request merged**